src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2405,6 +2405,7 @@ export default class AircraftModel {
         return shouldEnterHold;
     }
 
+    //TODO: can this be removed? It doesn't do anything.
```

Comment:
it actually does do stuff.  

This method is called via the `EventBus`: https://github.com/openscope/openscope/tree/develop/src/assets/scripts/client/aircraft/AircraftController.js#L172 which gets triggered in another class (likely the `AircraftController`)